src/main/java/mat/shared/model/util/MeasureDetailsUtil.java

```diff
@@ -94,4 +100,22 @@ public static String defaultTypeIfBlank(String type) {
         return type == null || type.isEmpty() ? PRE_CQL : type;
     }
 
+    public static boolean isValidatable(String releaseVersion, String qdmVersion, boolean draft, boolean draftable, String modelType) {
+        if (releaseVersion == null || qdmVersion == null || modelType == null) {
+            return false;
+        }
+        BigDecimal matVersion = null;
+        BigDecimal measureVersion = new BigDecimal(qdmVersion);
+
+        RegExp regExp = RegExp.compile("[0-9]+\\.[0-9]+");
```

Comment:
regexp compilation is considered to be expensive. you might want to cache it in a static field.